axum-extra/src/extract/cookie/mod.rs

```diff
@@ -255,7 +255,7 @@ mod tests {
                     custom_key: CustomKey(Key::generate()),
                 };
 
-                let app = Router::<_, Body>::new()
+                let app = Router::<_>::new()
```

Comment:
In expression position, generic parameter defaults don't apply so this can be

```suggestion
                let app = Router::new()
```